playbooks/roles/promtail/tasks/main.yml

```diff
@@ -14,7 +14,9 @@
     name: "{{ item }}"
   loop:
     - "{{ packages }}"
-  when: "ansible_facts.pkg_mgr != 'atomic_container'"
+  when: 
+  - "ansible_facts.pkg_mgr != 'atomic_container'"
+  - "inventory_hostname != 'bridge.eco.tsi-dev.otc-service.com'"
```

Comment:
uhm, this is not the nice way to solve this. We should rather fix docker on bridge

Reply:
solved by adding ```use_upstream_docker: False``` as host variable to bridge